### PR TITLE
Firefox support

### PR DESCRIPTION
**Pull request by kelonye**
Closes Teamweek/tw-button/issues/40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,3 +131,16 @@ If you are using Atom make sure you have installed these packages:
...
+
+# Firefox development
+
+Install [web-ext](https://github.com/mozilla/web-ext)
+
+    $ npm i -g web-ext
+
+Start watcher:
+
+    $ cd app && web-ext run
```
Comment (dhruvparmar372) [inline README.md]: @kelonye is there something similar for chrome?

**Comment by kelonye** [comment]
@dhruvparmar372 am working on https://blog.mozilla.org/addons/2016/05/31/using-google-analytics-in-extensions before merging .. since firefox disallows including external scripts.

File: README.md (hunk 1 above)
Comment (kelonye) [reply README.md]: they're working on adding chrome support but i'll investigate if there're existing similar tools

**Comment by kelonye** [comment]
Firefox branch will be maintained in parallel

**Pull request closed without merging**